### ISSUE TITLE
No tooltip for "run test" button

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#166327]
The button that runs a test should have a tooltip telling me what will happen when I click it. Otherwise there's actually no way to know what the button is.

<img width="65" alt="image" src="https://user-images.githubusercontent.com/323878/201798368-329c5c7c-fbeb-46ad-93ca-43589b300c43.png">

**Comment by roblourens** [comment]
The tooltip is nice because I didn't know I could right click there :)
